packages/forms/src/types/types.ts

```diff
@@ -73,7 +73,7 @@ export interface IInputDefinition<T = unknown> {
    * Note: Validation of complex inputs has to respect its formik structure
    */
   validation?: {
-    schema?: Schema<unknown>
+    schema?: (values: any) => Schema<unknown>
```

Comment:
looks like a breaking change for any view using the validation schema today.
1. either we should make the corresponding changes in the same PR
2. OR support both static and callback schemas `Schema<unknown> | (values: any) => Schema<unknown>`